public/index.html

```diff
@@ -5,10 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>DPdash</title>
-  </head>
+  <script defer src="/js/bundle.js"></script></head>
```

Comment:
Did you move `<script>` element from body to head? @Elkrival

Reply:
Webpack appends this during the build process, it recreates an index.html file using the template file and it routes to the built JavaScript.